Inseok_45/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "inseok_45",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dbmate": "^2.2.0",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "morgan": "^1.10.0",
+    "mysql2": "^3.2.3",
+    "nodemon": "^2.0.22",
```

```suggestion
    "mysql2": "^3.2.3",
```
nodemon, morgan 같은 경우는 dependencies 보다, npm uninstall 후 dev dependencies에 설치될 수 있도록 해주세요~